packages/amplify-schema-validator/src/index.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable global-require */
+
+import { parse } from 'graphql';
+
+const allValidators = [
+  require('./validators/index-scalar-types').validate,
+  require('./validators/relation-required-with-belongs-to').validate,
+  require('./validators/two-many-to-many-locations').validate,
+];
+
+export const validateSchema = (schemaString: string): boolean => {
+  const schema = parse(schemaString);
+  allValidators.forEach((validator) => {
+    validator(schema);
+  });
+  return true;
```

Comment:
We should either return a boolean, or throw an error but if we're rely on the error to communicate info, this and underlying `validator` should just have a void return.

Reply:
I'm not sure why, but in terms of test readability,

```
const result = validate(schema);
expect(result).toBe(true)
```

just looks so much cleaner to me than 

```
expect(() => validate(schema)).not.toThrow();
```

Reply:
From a testing perspective, I think you're probably right, but I think the decision to collect an array of error messages makes this point somewhat moot :D